projects/valtimo/shared/assets/core/nl.json

```diff
@@ -1743,19 +1743,19 @@
     "uiComponent": "Selecteer UI component"
   },
   "processLinkTypeDisabledTooltip": {
-    "plugin": "Er zijn geen plugin-configuraties beschikbaar. Maak een pluginconfiguratie aan om deze te kunnen koppelen aan een processtap.",
+    "plugin": "Er zijn geen plugin-configuraties beschikbaar. Maak een plugin-configuratie aan om deze te kunnen koppelen aan een processtap.",
     "form": "Er zijn geen formulieren beschikbaar. Maak een formulier aan om deze te kunnen koppelen aan een processtap.",
     "form-flow": "Er zijn geen FormFlows beschikbaar. Maak een FormFlow aan om deze te kunnen koppelen aan een processtap.",
-    "ui-component": "Er zijn geen custom componenten geregistreerd maak er een aan om deze optie te kunnen gebruiken."
+    "ui-component": "Er zijn geen maatwerk componenten geregistreerd maak er een aan om deze optie te kunnen gebruiken."
   },
   "processLinkUrlInvalid": "Ongeldige URL {{ URL }}",
   "accessControl": {
     "roles": {
       "title": "Alle rollen",
-      "key": "Rolsleutel",
+      "key": "Rol-sleutel",
```

Comment:
Geldt op veel plekken; overbodig streepje; rolsleutel is prima leesbaar en begrijpelijk